src/FixerConfiguration/FixerConfigurationResolver.php

```diff
@@ -63,7 +63,8 @@ public function resolve(array $options)
                 $alias = $option->getAlias();
 
                 if (array_key_exists($alias, $options)) {
-                    // @TODO 2.12 Trigger a deprecation notice and add a test for it
+                    @trigger_error(sprintf('Option %s is deprecated, use %s instead.', $alias, $name), E_USER_DEPRECATED);
```

Comment:
please use double quotes, like;  `'Option "%s" is deprecated, use "%s" instead.'`